Azure-Samples/azure-intelligent-edge-patterns
# Adding kubeflow-on-azure-stack section

## Purpose

* Adding the section about Kubeflow on Azure Stack. Step-by-step instructions to create Kubeflow cluster and start the dashboard.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
[follow instructions see that it results in functioning Kubeflow cluster]
```

* Test the code

```
Manual testing. Follow instructions see that it results in a functioning Kubeflow cluster instance.
```

## What to Check
Verify that the following are valid
* You can see the Kubeflow dashboard via the external ip.

## Other Information
